### PR TITLE
LPD-34325 | Test Fix | Poshi >>> Playwright 

### PR DESCRIPTION
**Pull request by rafaellajordao**
### Motivation:
Test failing due to Poshi limitations.

### Proposed Solution:
Transcribe test to playwright in order to facilitate maintenance.

**More details:** [LPD-34325](https://liferay.atlassian.net/browse/LPD-34325)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
> ### ❌ ci:test:analytics-cloud - 243 out of 255 jobs passed in 2 hours 49 minutes
> Click here for more details.

all the tests modified in this PR are passing correctly! :smile:

File: modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+
+			await expect(
+				page.getByText('There are no visitors data found.')
+			).toBeVisible();
+			await expect(
+				page.getByText(
+					'Check back later to verify if data has been received from your data sources, or you can try a different date range.'
+				)
+			).toBeVisible();
+			await expect(
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts:91]: @rafaellajordao can you remove this statement, the below one should already check if the element is visible or not

File: modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+				page.getByRole('link', {name: 'Learn more about blogs.'})
+			).toBeVisible();
+
+			await page
+				.getByRole('link', {name: 'Learn more about blogs.'})
+				.click();
+
+			const newPagePromise = page.waitForEvent('popup');
+
+			const newPage = await newPagePromise;
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts:101]: @rafaellajordao please inline

File: modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+
+			const href = newPage.url();
+
+			await expect(href).toContain('learn.liferay.com');
+			await expect(href).toContain('/optimization/a-b-testing');
+
+			await newPage.close();
+		});
+
+		await test.step('Delete channel', async () => {
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/emptyState.spec.ts:545]: @rafaellajordao please use afterEach for tearDown

**Review by marcosapmf — CHANGES_REQUESTED** [review]

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
> ### ❌ ci:test:analytics-cloud - 238 out of 255 jobs passed in 4 hours 37 minutes
> Click here for more details.

all the tests modified in this PR are passing correctly! :smile:

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**